website/content/en/scripts/step11-create-provisioner.sh

```diff
@@ -0,0 +1,26 @@
+cat <<EOF | kubectl apply -f -
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  requirements:
+    - key: karpenter.k8s.aws/instance-category
+      operator: In
+      values: [c, m, r]
+    - key: karpenter.k8s.aws/instance-generation
+      operator: Gt
+      values: ["2"]
+  providerRef:
+    name: default
+---
+apiVersion: karpenter.k8s.aws/v1alpha1
+kind: AWSNodeTemplate
+metadata:
+  name: default
+spec:
```

Comment:
WDYT about configuring instanceProfile here, since we're considering removing it from the global configmap in our next api version.

Reply:
Okay. Here's what I added (I got the example from another part of our docs):
`  instanceProfile: MyInstanceProfile # optional, if already set in controller args`